lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +193,18 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+@pytest.fixture(scope='function')
```

Comment:
Did you mean to make this a new fixture?  The coverage is failing because this is never getting run